test/prompt/prompt.go

```diff
@@ -51,7 +51,7 @@ func (pt *Prompt) RunTest(t *testing.T, procedure func(*goexpect.Console) error,
 	assert.NilError(t, test(stdio(c)))
 
 	// Close the slave end of the pty, and read the remaining bytes from the master end.
-	c.Tty().Close()
+	_ = c.Tty().Close()
```

Comment:
any reason for changing this line?

Reply:
Isn't that for errcheck?

Reply:
Yeah the `make check` failed on this line (and others like it). I wasn't sure if this was intentional or not, so I kept the functionality the same and suppressed the warning.